x/incentives/keeper/distribute.go

```diff
@@ -292,6 +292,12 @@ func (k Keeper) distributeInternal(
 		remainEpochs = gauge.NumEpochsPaidOver - gauge.FilledEpochs
 	}
 
+	// defense in depth
+	// this should never happen in practice since gauge passed in should always be an active gauge.
+	if remainEpochs == uint64(0) {
+		return nil, fmt.Errorf("gauge with id of %d is not active", gauge.Id)
+	}
```

Comment:
Should we also be adding a check in validate basic that `NumEpochsPaidOver` can't be 0?

Reply:
@p0mvn which validate basic are you referring to ?